torch/package/package_exporter.py

```diff
@@ -154,7 +154,25 @@ def __init__(self, dependency_graph: DiGraph):
                 error_context = dependency_graph.nodes[module_name].get("error_context")
                 if error_context is not None:
                     message.write(f"      Context: {error_context}\n")
-
+                if module_name in _DISALLOWED_MODULES:
+                    message.write(
+                    (
+                    f"      Note: While we usually use modules in the python standard library "
+                    f"from the local environment, `{module_name}` has a lot of system "
+                    f"level access and therefore can pose a security risk. We heavily "
+                    f"recommend removing `{module_name}` from your packaged code. However, if that "
+                    f"is not possible, add it to the extern list by calling "
+                    f"PackageExporter.extern(\"`{module_name}`\")\n"
+                    )
+                    )
+                if debug:
+                    module_path = dependency_graph.first_path(module_name)
+                    message.write(f"      A path to {module_name}: {' -> '.join(module_path)}")
+        if not debug:
```

Comment:
shouldn't this be in the for loop?

Reply:
This error message specifically isn't related to an individual module and is just advertising the debug functionality. It wouldn't make sense to stick it in the loop.